packages/test/local-server-tests/src/test/synchronousDataStoreCreation.spec.ts

```diff
@@ -3,7 +3,10 @@
  * Licensed under the MIT License.
  */
```

Comment:
i basically rewrote this test to split the parent and child datastore, and handle more cases, so i'd recommend split view, to really just review the latest.